qcodes/instrument/parameter.py

```diff
@@ -1364,6 +1364,13 @@ def get(self):
             else:
                 return state['value']
 
+    def get_timestamp(self):
```

Comment:
Would you be so kind to add a simple test for this in `test_parameter.py`? :)

Reply:
Done